packages/builders/scripts/vats/upgradeVaults.js

```diff
@@ -13,6 +13,12 @@ export const defaultProposalBuilder = async ({ publishRef, install }) =>
             '../bundles/bundle-vaultFactory.js',
           ),
         ),
+        governorRef: publishRef(
+          install(
+            '@agoric/governance/src/contractGovernor.js',
+            '../bundles/bundle-contractGovernor.js',
```

Comment:
Please add a test that this works.

I just ran into at least one case where it does not while working on dapp-orchestration-basics deployment testing. Note the lack of `.reset()` in...

https://github.com/Agoric/agoric-sdk/blob/442f07c8f0af03281b52b90e90c27131eef6f331/packages/deploy-script-support/src/coreProposalBehavior.js#L168-L170

Perhaps an easy 1-line fix. IOU an issue. Fixing it in this same PR seems fine.

Reply:
> Please add a test that this works.

Ah, now that I understand #10172, I get that by "works", you mean that it saves the installation to promise space. That had never been part of my mental model of what `publishRef(install()` did, and so most of the time, I manually populated `produce.instance.contract`. 
Now that I get what it's supposed to be doing, I'll ensure that the key matches the one used for the governance installation.

Reply:
commit [7b9e75f](https://github.com/Agoric/agoric-sdk/pull/10163/commits/7b9e75f6b1bb3bcd271ab8c78786a03786748fd6) uses `restoreRef()` to replace a bunch of manual fiddling with bundles and `install()` calls. The line in `coreProposalBehavior.js` is part of the solution.